jOOL/src/main/java/org/jooq/lambda/Agg.java

```diff
@@ -82,6 +82,57 @@
         return Collectors.reducing((v1, v2) -> v2);
     }
 
+    /**
+     * Get a {@link Collector} that takes the first <code>n</code> elements from a collection.
+     * <p>The final type of this {@link Collector} is {@link Seq} type.
+     * @param n     The number of elements to be taken
+     * @param <T>   The type of element
+     * @return      A {@link Seq} of type <code>T</code>
+     */
+    public static <T> Collector<T, ?, Seq<T>> taking(long n) {
+        return Collector.of(
+                (Supplier<LinkedList<T>>) LinkedList::new,
+                (l, v) -> {
+                    if (l.size() < n) {
+                        l.add(v);
+                    }
+                },
+                (l1, l2) -> {
+                    l1.addAll(l2);
+                    return l1;
+                },
+                l -> Seq.seq(l.stream())
+        );
+    }
+
+    /**
+     * Get a {@link Collector} that skip the first <code>n</code> elements of a collection.
+     * <p>This method will skip the first <code>n</code> elements, and start from the (n+1)th element.
+     * The final type of this {@link Collector} is {@link Seq} type.
+     * @param n     The number of elements to be skipped
+     * @param <T>   The type of element
+     * @return      A {@link Seq} of type <code>T</code>
+     */
+    public static <T> Collector<T, ?, Seq<T>> dropping(long n) {
+        long[] index = {0};
```

Comment:
I'd prefer a `Collector` that doesn't use any capturing state like this. The `Collector.supplier()` should be able to produce an `Accumulator` type.